### ISSUE TITLE
Provide nice UI to choose date time for filtering with campaign created date time

### ISSUE THREAD

**Issue by karuppiah7890** [freeCodeCamp/Mail-for-Good#132]
For now I had to manually type the whole date time. It's quite tedious

**Comment by karuppiah7890** [comment]
The below screen shot shows how Plivo uses a nice UI to choose a from date and to date for filtering call logs : 

![plivo from](https://user-images.githubusercontent.com/12808424/27010368-36f3e138-4ec0-11e7-967c-c707f30e247d.png)

**Comment by AndrewGHC** [comment]
We've leveraged react-bootstrap-table to implement a large amount of data handling in this app. It's possible that they've implemented a feature akin to this. If not, the best approach from my perspective would be to submit a PR to this library and implement it directly. Will review this in due course.

**Comment by karuppiah7890** [comment]
Yes, I did see that! I agree. But then, we are actually using version 3.0.0 beta (https://github.com/freeCodeCamp/Mail-for-Good/blob/master/package.json#L73), and the current version of react-boostrap-table is 3.4 from what I see in their package.json in master branch, but then, they have 4.0.0 beta too now

**Comment by karuppiah7890** [comment]
We can choose one of them, but choosing 4.0 will have breaking changes for us

**Comment by AndrewGHC** [comment]
Just out curiosity, does 4.0 have any functionality in this regard that's worth upgrading for?

**Comment by karuppiah7890** [comment]
I haven't checked anything yet. I was planning on using another library, a bower component, but yes, I will check if React Bootstrap itself has any options for this UI
